app/models/sample.rb

```diff
@@ -334,11 +334,6 @@ def default_background_id
     host_genome && host_genome.default_background ? host_genome.default_background.id : Background.find_by(public_access: 1).id
   end
 
-  def as_json(options = {})
-    options[:methods] ||= [:input_files, :host_genome_name, :private_until]
```

Comment:
As said above, this only adds these methods if no methods are passed.
I am not agains removing this... But be careful with possible side effects.

Reply:
Yeah it's a little bit scary... I will test and be prepared to fight any fires that result :D